doc/lua_api.md

```diff
@@ -3659,6 +3659,9 @@ Player Inventory lists
     * Is not created automatically, use `InvRef:set_size`
     * Is only used to enhance the empty hand's tool capabilities
 
+Custom lists can be added via `minetest.set_size("name", size)`. Setting `size`
+to `0` deletes a list
```

Comment:
1. Does not only apply to players but to all instances of `InvRef`. For non-player `InvRev`, simply all lists will be custom. I don't think we need to repeat it here.
2. `minetest.` should be `InvRef:`